stable/nfs-server-provisioner/values.yaml

```diff
@@ -16,7 +16,7 @@ service:
 
   nfsPort: 2049
   mountdPort: 20048
-  rpcbindPort: 51413
+  rpcbindPort: 111
```

Comment:
Don't you need root/privileged access in order to open low-ports (<1024?)

Reply:
Hi, the nfs-server-provisioner was not to designed run as non root. It also requires some additional capabilities for make possible use VFS:

https://github.com/helm/charts/blob/bc0f5d09fdfcbd1d18575c0dc6da72f5893f8794/stable/nfs-server-provisioner/templates/statefulset.yaml#L50-L54

However you don't need any extra capabilities for bind on 111 port as root user.

Any way the upstream project uses `111` port for rpcbind:
https://github.com/kubernetes-incubator/external-storage/blob/2ba1b775cefa4b7393d805f7b98812c62eb1c167/nfs/deploy/kubernetes/statefulset.yaml#L34-L35